constants/destinationCanonicalNames.js

```diff
@@ -101,7 +101,8 @@ const DestCanonicalNames = {
     "Gainsight PX",
     "gainsight px",
     "Gainsight Px"
-  ]
+  ],
+  ga4: ["GA4", "ga4", "Ga4"]
```

Comment:
```suggestion
  ga4: ["GA4", "ga4", "Ga4", "Google Analytics 4", "googleAnalytics4"]
```
We need to support these kinds of names as well because in the doc we are mentioning this destination as google analytics 4.

Reply:
updated